inductiva/storage/storage.py

```diff
@@ -0,0 +1,89 @@
+"""Methods to interact with the user storage resources."""
+import inductiva
+from inductiva.client.apis.tags import instance_api
+from inductiva.utils import format_utils
+from typing import Literal
+
+
+def get_space_used():
+    """Returns the occupied storage size in GB."""
+    try:
+        api = instance_api.InstanceApi(inductiva.api.get_client())
+        response = api.get_storage_size()
+        storage_used = round(float(response.body), 3)
+        print(f"Total storage used: {storage_used} GB")
+        return storage_used
+    except inductiva.client.ApiException as api_exception:
+        raise api_exception
+
+
+def listdir(max_results: int = 3,
+            order_by: Literal["size", "creation_time"] = "size",
+            sort_order: Literal["asc", "desc"] = "desc"):
+    """List and display the contents of the user's storage bucket.
+    Args:
+        max_results (int): The maximum number of results to return.
+        order_by (str): The field to sort the contents by.
+        sort_order (str): Whether to sort the contents in ascending or
+        descending order.
+    Returns:
+        list of dict: A list of dictionaries containing information about 
+        the size, the name and the creation time of each content that can
+        easily be converted to a dataframe. 
+
+    This function prints a table with the storage content information: 
+        Name            Size            Creation Time
+        1234            5.68 MiB        29 Sep, 14:12:00
+        12345           374.85 KiB      29 Sep, 14:13:10
+        1234567         97.59 KiB       29 Sep, 14:13:24
+        123             0 B             29 Sep, 14:13:29
+        123456          0 B             29 Sep, 14:13:18
```

Comment:
Maybe you can add here that the users can use the dir information in the task class since they are the same and navigate through each task output.